roles/openshift_examples/tasks/main.yml

```diff
@@ -36,6 +36,7 @@
   unarchive:
     src: "{{ copy_examples_mktemp.stdout }}/openshift-examples.tar"
     dest: "{{ examples_base }}/"
+    mode: 0755
```

Comment:
Does this impact the permissions on the files unarchived and set them to 0755? if so, it might make sense to use: `mode: u=rwX,g=rX,o=rX` instead.